modules/text-input/react/lib/TextInput.tsx

```diff
@@ -46,6 +44,13 @@ const Input = styled('input')<Pick<TextInputProps, 'error' | 'grow'>>(
   ({error}) => ({
     ...errorRing(error),
   }),
+  ({width}) => ({
+    minWidth: width || 280,
+    width: width || 280,
+  }),
+  ({height}) => ({
+    height: height || 40,
+  }),
```

Comment:
I wanted to be able to customize the width/height of my text input box, so I added an optional height/width to Input

Reply:
Not sure if this will be approved by the design system team, this isn't something we're customizing at all since it the height of the inputs are what help the forms align together.

Reply:
how should I go about making the `goTo` input match the height/width of the spec?

Reply:
@lychyi @NicholasBoll just want to confirm you are okay with this change

Reply:
I'll have to try it out and involve Design System designers if necessary. I don't understand the context yet

Reply:
Figma has been updated with the 40px height `TextInput`s. Let's revert these changes for now while the team figures out what we want to do with "compact" style inputs across the board.

I don't think it's a good idea to augment the API for `TextInput` without having a holistic version across all input components.

Reply:
I agree

Reply:
Figma's updated. I'm reverting height changes.